app/services/push_service.rb

```diff
@@ -20,20 +20,44 @@ def notify
     PusherJob.perform_later([channel].flatten, event, data.to_json)
   end
 
+  def send_update_store(channel, data, collapse_key = nil)
+    @channel = channel
+    @event = "update_store"
+    @data = data
+    notify
+
+    channel = [channel].flatten.find_all{|c| Channel.user_channel?(c)}
+    unless channel.empty?
+      entity_type = nil, entity_id = nil
+      if collapse_key.nil?
+        entity_type = data[:item].object.class.name
+        entity_id = data[:item].id
+      else
+        # collapse_key should be in the format of "#{entity_type}#{entity_id}"
+        entity_id = collapse_key.scan(/\d+/).first.to_i
+        entity_type = collapse_key.sub(collapse_key.to_s, "")
+      end
+      operation = data[:operation].to_s
+      data = {event:'update_store', entity_type:entity_type, entity_id:entity_id, operation:operation, date:Time.now.to_json.tr('"','')}
```

Comment:
Line is too long. [136/80]<br>Space missing after colon.<br>Space missing after comma.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.